io.mrarm.mcpelauncher-ui.json

```diff
@@ -0,0 +1,109 @@
+{
+    "id": "io.mrarm.mcpelauncher-ui",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
```

Comment:
Update runtime to 5.14
```suggestion
    "runtime-version": "5.14",
```

Reply:
Ok runtime updated

Reply:
But now qtwebengine isn't found.

Reply:
This commit breaks kde runtime 5.13+ https://github.com/KDE/flatpak-kde-runtime/commit/fb5bbe40a1fc0406e101b1c9bfbe6262e0aa920d